SPECS-EXTENDED/whois/whois.spec

```diff
@@ -13,7 +13,7 @@ Distribution:   Mariner
 
 Name:       whois       
 Version:    5.5.7
-Release:    2%{?dist}
+Release:    3%{?dist}
```

Comment:
I see some PR checks failing. The reason is not related to your change, but it looks like the easiest way to resolve them would be to update this package to 5.5.15, which has an existing source URL. I've checked that the new version builds without issues locally, so you'd only need to do this:

- Change the `Version` tag to `5.5.15`.
- Modify the `%changelog` entry to reflect the new version.
- Add two more entries in the changelog for this PR. One saying that the version was updated and the other saying `Licence verified`. Exact wording is crucial! I've already checked the license for you, this is now just needed to resolve one of the PR checks.
- Update the `downloadUrl` field for `whois` inside the `[repo_root]/cgmanifest.json` file. Change it to `http://ftp.debian.org/debian/pool/main/w/whois/whois_5.5.15.tar.xz`.
- Same goes for the `version` field - it needs to be updated to `5.5.15`.

Sorry for the trouble! I couldn't update the PR from my machine - I didn't have access to your fork where the branch is.